backend/factory.ts

```
@@ -0,0 +1,36 @@
+  formatter,
+  (await getSettings()).travelSettings.distanceRefunds,
+  async (id: _id) => {
+    const doc = await DocumentFile.findOne({ _id: id }).lean()
+    if (doc) {
+      return { buffer: doc.data.buffer, type: doc.type }
+    }
+    return null
+  },
+  async (id: _id) => {
+    const orga = await Organisation.findOne({ _id: id }).lean()
+    if (orga && orga.logo && orga.logo._id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)